### PR TITLE
Fix code display in UI and verification checks against HMAC values.

### PR DESCRIPTION
**Pull request by mikehelmick**
## Proposed Changes

* Return plain text verification codes (short/long) to controller instead of hmac version
* On expiration check both plaintext and hmac version when matching for expiration date

**Release Note**

```release-note
BUG FIX - check expiration correctly after verification code HMAC upgrade.
```

### CODE REVIEW

**Comment by icco** [comment]
Hmmm, not sure why that test is failing...

File: pkg/database/vercode.go [hunk 1]
```diff
@@ -83,15 +83,23 @@ func (v *VerificationCode) FormatSymptomDate() string {
 
 // IsCodeExpired checks to see if the actual code provides is the
 // short or long code and deteriminies if it is expired based on that.
-func (v *VerificationCode) IsCodeExpired(code string) bool {
+func (db *Database) IsCodeExpired(v *VerificationCode, code string) (bool, error) {
+	hmacedCode := code
+	if len(code) < 20 {
```
Comment (MushuEE) [inline pkg/database/vercode.go:88]: Should we classify `20` with a name as a global? I am not really sure what `20` means. Is this < "short"?

Comment (mikehelmick) [reply pkg/database/vercode.go:88]: it was the previous max plaintext code length

can pull out to constant - or drop this and hmac the hmac value

**Pull request merged**